notebooks/stage/watson_keywords.ipynb

```diff
@@ -0,0 +1,1355 @@
+{
```

Comment:
<p>I think the prints here deserve a function. Something like:</p><p> </p><pre class="ql-syntax" spellcheck="false">from pprint import pprint

def describe_result(heading, obj):
    print(f'"{heading}"\n')
    pprint(obj, indent=3)
</pre><p> </p><p>Or you can even consider printing it as a table from a pandas data frame... The data is fortunately consistent, so pandas should be able to auto-format it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/mailing-list-analysis-toolkit/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>